### PR TITLE
Split favicon into two separate examples

### PR DESCRIPTION
**Pull request by AmySteam**
Creates two examples of how to use the favicon API on a popup (extension page) and on a content script

### CODE REVIEW

File: examples/favicon-cs/content.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+console.log("content-script")
```
Comment (oliverdunk) [inline examples/favicon-cs/content.js:1]: Should we remove this before merging?

File: examples/favicon-cs/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+See [Fetching favicons](https://developer.chrome.com/docs/extensions/mv3/favicon) to learn more. 
+
+## Testing the extension
+
+1. Follow the instructions to load an [unpacked extension](https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked).
+2. Navigate to [www.example.com](https://www.example.com/).
+
+It should look like this:
+
+<img src="https://wd.imgix.net/image/BhuKGJaIeLNPW9ehns59NfwqKxF2/3Q1glvnzbWhraXRtnGOy.png" alt="Content script using the Favicon API" width="400"/>
```
Comment (oliverdunk) [inline examples/favicon-cs/README.md:16]: It looks like this is our first time using imgix, are we comfortable hosting there?

**Review by oliverdunk — CHANGES_REQUESTED** [review]
Tested both examples and they're working great, just left a couple of nits :)

File: examples/favicon-cs/content.js (hunk 1 above)
Comment (AmySteam) [reply examples/favicon-cs/content.js:1]: Yep! Missed that 😅

File: examples/favicon-cs/README.md (hunk 2 above)
Comment (AmySteam) [reply examples/favicon-cs/README.md:16]: Oh right, I forgot to mention this is where all images of https://developer.chrome.com/ are hosted.
I don't think this is a blocker, but it's probably something we should agree on with the team. Thanks for bringing this up 👍

File: examples/favicon-cs/manifest.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      "js": ["content.js"],
+      "css": ["style.css"]
+    }
+  ],
+  "permissions": ["favicon"],
+  "web_accessible_resources": [
+    {
+      "resources": ["_favicon/*"],
+      "matches": ["<all_urls>"],
+      "extension_ids": ["*"]
```
[the remark this answers is not in the document]
Comment (AmySteam) [reply examples/favicon-cs/manifest.json:17]: Will do!

**Review by oliverdunk — APPROVED** [review]
LGTM 👍

**Pull request merged**